### PR TITLE
2403

### PR DESCRIPTION
**Pull request by EightSmart**

### CODE REVIEW

**Review by Syntax505 — APPROVED** [review]
gud

File: Data/Sector/Hong-Kong-Sector-File.ese [hunk 1]
```diff
@@ -1692,54 +1706,54 @@ SECTORLINE:77 ;Not drawn
...
+DISPLAY:VHHK·VHHK_SCT_TRS_01·000·250:VHHK·VHHK_SCT_TRS_01·000·250:VHHK·VHHK_SCT_TRC_01·000·250
+DISPLAY:VHHK·VHHK_SCT_TRS_01·250·365:VHHK·VHHK_SCT_TRS_01·250·365:VHHK·VHHK_SCT_TRC_01·250·365
+DISPLAY:VHHK·VHHK_SCT_TRC_01·000·250:VHHK·VHHK_SCT_TRC_01·000·250:VHHK·VHHK_SCT_TRS_01·000·250
+DISPLAY:VHHK·VHHK_SCT_TRC_01·250·365:VHHK·VHHK_SCT_TRC_01·250·365:VHHK·VHHK_SCT_TRS_01·250·365
 
-SECTORLINE:80 ;TRW1 East
+SECTORLINE:80 ;TRV East
 COORD:N020.50.17.621:E114.12.24.030
 COORD:N019.59.43.496:E114.08.57.157
 COORD:N019.49.12.585:E113.58.42.622
```
Comment (Syntax505) [inline Data/Sector/Hong-Kong-Sector-File.ese:1731]: This coordinate seems to be out of place?

**Review by Syntax505 — CHANGES_REQUESTED** [review]

File: Data/Sector/Hong-Kong-Sector-File.ese (hunk 1 above)
Comment (Syntax505) [reply Data/Sector/Hong-Kong-Sector-File.ese:1731]: wasn't highlighted correctly for some reason, specifically line 1731

**Review by Syntax505 — APPROVED** [review]

**Review by UraniumLover — APPROVED** [review]
ok thx

**Pull request merged**